automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoApiClient.scala

```diff
@@ -40,10 +42,12 @@ object LeonardoApiClient {
   implicit def getRuntimeDoneCheckable[A]: DoneCheckable[GetRuntimeResponseCopy] =
     (op: GetRuntimeResponseCopy) => op.status == ClusterStatus.Running || op.status == ClusterStatus.Error
 
-  val client: Resource[IO, Client[IO]] = for {
-    blockingEc <- ExecutionContexts.cachedThreadPool[IO]
-    client <- blaze.BlazeClientBuilder[IO](blockingEc).resource
-  } yield Logger[IO](logHeaders = true, logBody = true)(client)
+  def client(implicit timer: Timer[IO]): Resource[IO, Client[IO]] =
+    for {
+      blockingEc <- ExecutionContexts.cachedThreadPool[IO]
+      retryPolicy = RetryPolicy[IO](RetryPolicy.exponentialBackoff(30 seconds, 5))
```

Comment:
this should make those `org.http4s.blaze.pipeline.Command$EOF$: EOF` much less often